roles/openshift_version/meta/main.yml

```diff
@@ -12,6 +12,7 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
+- role: openshift_inventory
```

Comment:
I see, normalization is different than validation in that it mutates the inventory variables, validation would imply no changes... `openshift_invetory_preprocessing`?! idk...